tests/app.test.js

```diff
@@ -0,0 +1,102 @@
+const app = require("../routes/app.js");
+const supertest = require("supertest");
+const { router } = require("../routes/urlshortner.js");
+const request = supertest(router);
+const mainServerRequest = supertest(app);
+const fs = require("fs");
+const { describe, it } = require("@jest/globals");
+const testUrl =
+  "https://www.google.com/search?q=adadsa&oq=adadsa&aqs=chrome..69i57j0i10l9.708j0j7&sourceid=chrome&ie=UTF-8";
+const alreadyShortenedUrl =
+  "https://stackoverflow.com/questions/48248832/stylesheet-not-loaded-because-of-mime-type";
+let oldCount;
+let oldShortID;
+
+describe("URL SHORTENER API", () => {
```

Comment:
Missing tests:
`404` and `400` for both `statistic` route and `GET` endpoint in `shorturl`